.github/workflows/cancel.yml

```diff
@@ -0,0 +1,17 @@
+name: Cancel
+on:
+  pull_request_target:
+    types:
+      - edited
+      - synchronize
+
+jobs:
+  cancel:
+    name: 'Cancel Previous Runs'
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    steps:
+      - uses: styfle/cancel-workflow-action
+        with:
+          workflow_id: 46388772, 34612818
```

Comment:
What are these two ids refer to?

Reply:
https://api.github.com/repos/toeverything/blocksuite/actions/workflows